### DISCUSSION TITLE
OpenTelemetry + AzureMonitorExporter + Recording Exception - missing stack trace

### DISCUSSION THREAD

**Discussion by michalwawa** [open-telemetry/opentelemetry-dotnet#3280] [Q&A]
Hello

Im currently using OpenTelemetry with AzureMonitorExporter to send logs to Application Insights. The problem is that when im using TelemetrySpan with extension RecordException, stack trace is not available in Insights. I can only see basic information about the event/log but there is no information what cause the problem.

Example of my code:

```
 using var activeSpan = _tracer.StartActiveSpan("Job");

try{
  do something.
}
catch (Exception ex)
{
   activeSpan.SetStatus(Status.Error);                
   activeSpan.SetAttribute("exception.stacktrace", ex.ToString()); -- only because of this i got in custom propery information about error
   activeSpan.RecordException(ex);
   context.Response.StatusCode = (int)HttpStatusCode.InternalServerError;
}

```
This is missing in insights view:
![image](https://user-images.githubusercontent.com/92430838/168251530-9d514d60-753d-47f4-bb8d-e3022fccb19c.png)

This is the view when im using custom property:
![image](https://user-images.githubusercontent.com/92430838/168252061-18f69c88-007b-44b8-9841-a2b39bfd280a.png)

Anyone know what can be a problem with this ?

**Comment by p10tyr** [comment]
I am not 100% sure but if you are using AzureMonitorExporter that should be picking up on the exceptions by a configured instrumentation? 
For example HttpInstrumnetation . if that throws an exception then that will transform the data to the required OTLP model and export it to Azure.
I have the similar issue Elastics Open Telemetry protocol. I cant just craft my own attributes .. it only seems to work when things are instrumented

**Comment by cijothomas** [comment]
https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter - Use this repo for AzureMonitorExporter questions. Its not owned by OpenTelemetry/CNCF.
